packages/components/__mocks__/day-calendar.js

```diff
@@ -24,7 +28,7 @@ class DayCalendar extends React.Component {
 								const day = getDate(date);
 								return (
 									<td key={dayIndex}>
-										<button onKeyDown={event => onKeyDown(event, this.calendarRef, day - 1)}>
+										<button id={this.isCurrentMonth(date) && day} onKeyDown={event => onKeyDown(event, this.calendarRef, day - 1)}>
```

Comment:
this id look very wick, and even if we add prefix to it like `week-1` It means you can t have multiple calendar on the same page. Don t know if this acceptable.

Reply:
make sense, will change it.

Reply:
use name for now